diff --git a/.vuepress/config.ts b/.vuepress/config.ts
index 9f3a828..821a729 100644
--- a/.vuepress/config.ts
+++ b/.vuepress/config.ts
@@ -13,5 +13,6 @@ import { html5Media as mdItHtml5Media } from 'markdown-it-html5-media';
 
 import sections, { sidebars } from './sections';
 import { imageSizeAttributesPlugin } from './plugins/image-size-attributes';
+import { nextSearchPlugin } from 'vuepress-plugin-next-search';
 
 const __dirname = getDirname(import.meta.url);
@@ -144,6 +145,14 @@ export default defineUserConfig({
 			imgMark: true,
 			flowchart: true,
 		}),
+		nextSearchPlugin({
+			fullText: true,
+			placeholder: '搜索',
+			frontmatter: {
+				tag: '标签',
+				category: '分类',
+			},
+		}),
 		imageSizeAttributesPlugin,
 		sitemapPlugin({
 			// The sitemap plugin from theme is somehow broken
diff --git a/package.json b/package.json
index 8d29a33..29852ce 100644
--- a/package.json
+++ b/package.json
@@ -10,4 +10,5 @@
     },
     "license": "SEE LICENSE IN LICENSE",
     "devDependencies": {
+        "vuepress-plugin-next-search": "file:../vuepress-plugin-next-search-fix",
         "@types/node": "^20.11.18",
